wordpress-mobile/WordPress-iOS
# Media Picker: Prevent crash if no error is provided when picking media

Fixes #14958. Associated Media Picker PR: https://github.com/wordpress-mobile/MediaPicker-iOS/pull/360

**To test**

- Build on iOS 14
- Visit the Media Library, choose +, tap Choose From My Device, then Select More Photos (or Select Photos if this is the first time you've run the app). Ensure that no photos are selected and tap Done.
- The media picker should be displayed with no content, and the app should not crash.

PR submission checklist:
